dns/octodns-config-validate.yaml

```diff
@@ -0,0 +1,74 @@
+# This file is copy of octodns-config.yaml file except the gcp section under provider for the validation purpose.
+# Update the octodns-config.yaml file likewise if any changes are made to this file.
+---
+providers:
+  config:
+    class: octodns.provider.yaml.YamlProvider
+    directory: ./zone-configs
+    enforce_order: false
+    default_ttl: 3600
+
+zones:
+  # Canary targets.
+  canary.kubernetes.io.:
+    sources:
+      - config
+    targets:
+      - gcp
```

Comment:
These all refer to a provider that doesn't exist - is that OK?

Reply:
For the validate it didn't consider this entry though..

Reply:
It just worked.. even I was surprised..